plugins/repository-hdfs/build.gradle

```diff
@@ -152,20 +152,29 @@ for (String integTestTaskName : ['integTestHa', 'integTestSecure', 'integTestSec
     runner {
       onlyIf { BuildParams.inFipsJvm == false }
       if (integTestTaskName.contains("Ha")) {
+        Path portsFile
+        File portsFileDir = file("${workingDir}/ports")
         if (integTestTaskName.contains("Secure")) {
-          Path path = buildDir.toPath()
+          portsFile = buildDir.toPath()
             .resolve("fixtures")
             .resolve("secureHaHdfsFixture")
             .resolve("ports")
-          nonInputProperties.systemProperty "test.hdfs-fixture.ports", path
         } else {
-          Path path = buildDir.toPath()
+          portsFile = buildDir.toPath()
             .resolve("fixtures")
             .resolve("haHdfsFixture")
             .resolve("ports")
-          nonInputProperties.systemProperty "test.hdfs-fixture.ports", path
         }
-        classpath += files("$buildDir/fixtures")
+        nonInputProperties.systemProperty "test.hdfs-fixture.ports", file("$portsFileDir/ports")
```

Comment:
not hugely important, but the path here will be "${workingDir}/ports/ports" right? Maybe name the file "hdfsPorts" or something like that

Reply:
I've renamed this directory `hdfsFixture` to be most accurate.